### DISCUSSION TITLE
missing snd-aloop module?

### DISCUSSION THREAD

**Discussion by paalj69** [Joshua-Riek/ubuntu-rockchip#403] [Q&A]
I am trying to install camilladsp on a radxa 5b and seems to be missing an alsa loopback function. Is this module available in any packages?

**Comment by StuartIanNaylor** [comment]
Is it complaining of no module after a modprobe as likely the kernel option has not been enabled?
https://sysplay.in/blog/tag/snd-aloop-driver/
